runner/utils.go

```diff
@@ -25,12 +25,14 @@ func isTmpDir(path string) bool {
 
 func isIgnoredFolder(path string) bool {
 	paths := strings.Split(path, "/")
-	if len(paths) <= 0 {
+	pathsOnWindows := strings.Split(path, "\\")
```

Comment:
it does not work on Window (for  `\`)

![image](https://cloud.githubusercontent.com/assets/12199042/16733964/64d14a5a-478c-11e6-8a2d-88f99fc827aa.png)